src/shop-app.html

```diff
@@ -250,6 +251,18 @@
         num-items="{{numItems}}"
         total="{{total}}"></shop-cart-data>
 
+    <!--
+      shop-payment-request provides the logic and hooks into the PaymentRequest API.
+    -->
+    <shop-payment-request
+        page="[[page]]"
```

Comment:
Generalized it a bit, following your suggestion. A quick followup could be (though I haven't yet) to pull out `_updateShippingAddress` into an app-specific callback, similar to `process-payment` as it has a lot of app-specific logic (i.e., logic around shipping costs).